js/index.js

```diff
@@ -1,23 +1,35 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  //console.log('Calculating subtotal, yey!');
+  const price = product.querySelector('.price span').textContent;
+  const quantity = product.querySelector('.quantity input').value;
+  //console.log(`quantity is ${quantity}`);
+  const total = (price * quantity);
+  //console.log(`price is ${price}`);
+  //console.log(`${price} ${quantity} my total is ${total}`);
+  const subTotal = product.querySelector('.subtotal span');
+  return subTotal.innerHTML = total;
 }
 
 function calculateAll() {
-  // code in the following two lines is added just for testing purposes.
-  // it runs when only iteration 1 is completed. at later point, it can be removed.
   const singleProduct = document.querySelector('.product');
   updateSubtotal(singleProduct);
-  // end of test
-
-  // ITERATION 2
-  //... your code goes here
-
+  
+  let totalValue = 0; 
+
+  const cartItemElements = document.getElementsByClassName('product');
+  for (cartItemElement of cartItemElements){
+    totalValue += updateSubtotal(cartItemElement)
+  }
+  
+  console.log(totalValue)
```

Comment:
I'd recommend removing `console.log()` once you have confirmed that you're getting the right values, it doesn't serve a purpose afterwards